lib/backend_scripts/primer-monitor.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+if ! ( set -o noclobber; : > "$BACKEND_SCRATCH_PATH/status/summarize_variants_running.lock" ) &> /dev/null; then
```

Comment:
The nextflow process is already not qsubbed, so I could put that in this release too, good idea.

Reply:
I've done this for the summarize_variants and recall_pangolin pipelines, the process-primer-sets one is a little more complex because it only conflicts with _part_ of summarize_variants.